Essentials/src/messages.properties

```diff
@@ -429,21 +413,20 @@ pTimeSetFixed=\u00a76Player time is fixed to \u00a7c{0}\u00a76 for\: \u00a7c{1}.
 pWeatherCurrent=\u00a7c{0}\u00a76''s weather is\u00a7c {1}\u00a76.
 pWeatherInvalidAlias=\u00a74Invalid weather type
 pWeatherNormal=\u00a7c{0}\u00a76''s weather is normal and matches the server.
-pWeatherOthersPermission=\u00a74You are not authorized to set other players'' weather.
+pWeatherOthersPermission=\u00a74You are not authorized to set other players weather.
```

Comment:
I don't think it should be the double '' either, but I still think proper english dictates that there is a single ' at the very end since we are changing multiple other players weather that they maintain ownership over.

idk let me ask my wife she knows these things

Reply:
Wife got back to me, since this would be a 'plural posessive', it should only contain one.

Reply:
Yes, but if only have one, the file will crash and the plugin will make a bug at startup so it will not work.
So the right word would be the `player's`?

Reply:
Oh I see now. You have to put in the '' twice for it to show up once.

This is correct as is - your change isn't grammatically correct.